.github/ISSUE_TEMPLATE.md

```diff
@@ -29,24 +29,40 @@
 ### Environment
 
 <!--
-  Please fill in all the relevant fields by running these commands in terminal.
+  If you are on v0.48.0 or greater, please run `react-native info` and paste its contents here then specify target platform, like this:
```

Comment:
We don't want to encourage people to open issues for old releases, and 0.48 is already the current stable. Can you rewrite this a bit so that it assumes people are using 0.48 or newer? Just have them run `react-native info`, and maybe add a note saying that if they get (insert whatever error is shown if they're not using 0.48+) then they should make sure they're using a recent version of React Native.

That way we can remove the whole next section that asks people to run several commands.

Reply:
Sounds good to me. Will update.